net/pfSense-pkg-pfBlockerNG/files/usr/local/www/pfblockerng/pfblockerng_alerts.php

```diff
@@ -647,7 +652,11 @@ function conv_log_filter_lite($logfile, $nentries, $tail, $pfbdenycnt, $pfbpermi
 
 	if (file_exists($logfile)) {
 		// Collect filter.log entries
-		exec("/usr/local/sbin/clog {$logfile} | {$pfb['grep']} -v '\"CLOG\"\|\"\033\"' | {$pfb['grep']} 'filterlog:' | /usr/bin/tail -r -n {$tail}", $logarr);
+		if (substr(trim(file_get_contents('/etc/version')), 0, 3) > '2.5') {
+			exec("{$pfb['grep']} -v '\"CLOG\"\|\"\033\"' {$logfile} | {$pfb['grep']} 'filterlog:' | /usr/bin/tail -r -n {$tail}", $logarr);
```

Comment:
See above comments about `dump_log()` but also should you choose to keep going this way, you can remove the initial grep -v that removes `CLOG` and the other related value -- they cannot be present on 2.5.0 so filtering them is no longer useful.